populator-machinery/controller.go

```diff
@@ -701,9 +722,21 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 				annSelectedNode: nodeName,
 			}
 		}
+		if c.mutatorConfig != nil && c.mutatorConfig.PvcPrimeMutator != nil {
+			mp := MutatorParams{
+				Pvc:          pvc,
+				PvcPrime:     pvcPrime,
+				StorageClass: storageClass,
+			}
+			err = c.mutatorConfig.PvcPrimeMutator(mp)
+			if err != nil {
+				c.recorder.Eventf(pvc, corev1.EventTypeWarning, reasonPVCPrimeMutatorError, "Failed to mutate populator PVC prime: %s", err)
```

Comment:
This mutates more than just the PVC prime, right?

Reply:
With the PvcPrimeMutator function it should only modify the PVC prime. I updated the function definition as we discussed above to avoid confusion